website/docs/components/form/masked-input/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Masked Input
+description: A form input that visually obfuscates characters to protect sensitive information by replacing them with a shape.
+caption: A form input that obfuscates sensitive information.
+links:
+  figma: {link to the "Form::MaskedInput" page in the components Figma library}
```

Comment:
@cveigt this is missing

Reply:
It's missing because the branch hasn't been merged, so there's no page/node to point to in Figma yet